taskcluster/xpi_taskgraph/transforms/release_github.py

```diff
@@ -74,7 +74,7 @@ def build_worker_definition(config, jobs):
             "version": config.params["version"],
             "build_number": config.params["build_number"],
         }
-        tag_name = manifest_config.get("release-tag", "{version}").format(
```

Comment:
I'm not sure.  This is already in the `ship` phase, so as it is there's not really an opportunity to have "subsequent" tasks.
I guess one option would be to move this to the `promote` phase, make it a pre-release, and have the `ship` phase push a `{version}` tag/release?  But then I don't think anything stops us making a build3 and breaking things again, especially since we don't bump the version when we "ship".

Reply:
Got it, thanks for the context! If I understand correctly, in a glorious future, we would have one task in the `promote` phase, another in the `ship` one + a version bump task in the `ship` phase. This way, we would be closer to what we have in Firefox.